contrib/misc/ft_laggedcoherence.m

```diff
@@ -169,10 +183,18 @@
   cfg_freq.foi = freq(freqindx);
   cfg_freq.t_ftimwin = t_ftimwin(freqindx);
   cfg_freq.toi = toicell{freqindx};
-  cfg_freq.pad = ceil(numel(timevec)./Fs.*cfg_freq.foi)./cfg_freq.foi;
+%   cfg_freq.pad = ceil(numel(timevec)./Fs.*cfg_freq.foi)./cfg_freq.foi;
+%
+% Code for checking whether the requested cfg_freq.t_ftimwin (calculated as t_ftimwin) allows for the
+% requested cfg_freq.foi (calculated as freq) using the requested number of cycles (cfg.numcycles).
+% 
+% cyclelengths=t_ftimwin/cfg.numcycles
+% correspondingfreqs=ones(size(cyclelengths))./cyclelengths
+% freq % compare with correspondingfreqs
+
   freqout{freqindx} = ft_freqanalysis(cfg_freq,datain);
   
-  cfg_lcoh.laggedcoherence.lags = cfg.numcycles.*cfg.loi./freqout{freqindx}.freq;
+fr  cfg_lcoh.laggedcoherence.lags = cfg.numcycles.*cfg.loi./freqout{freqindx}.freq;
```

Comment:
I guess that the 'fr' is a typo?

Reply:
Met dikke vingers getikt, maar het punt dat ik wilde maken betreft de code die daarvoor staat ...